### DISCUSSION TITLE
WETH -> wWETH Transaction not working correctly.

### DISCUSSION THREAD

**Discussion by Citizen-000** [wormhole-foundation/wormhole#445] [Problems / bugs] [answered]
Hello. I attemped a transaction from the ETH network to SOL network (Metamask to Phantom Wallet). The transaction supposedly completed, but the wWeth did not appear in the Phantom wallet. I tried a smaller amount afterwards, which DID go through, but the first transaction did not.

First transaction:
https://explorer.solana.com/tx/3C8kYeFw9w7j3hDJT5mbxsrZsK4G61DMnAyT55WhPHTLJr7C2yUnqriSVVH4HjP9dFgqash33hUV7apc68tBvGDi

Second transaction(completed):
https://explorer.solana.com/tx/67VMWiFPdDwbEzSbLD93LLiQUCWQTZt72TXvijcmYWFr1nCqWozhzo6NkHFNodtQ3fGC1Crsp5ktB46akYfeXS3D

**Comment by leoluk** [comment, marked as answer]
The first transaction you provided did increase your wWETH token balance.

The second transaction is a token account creation.

Please provide the ETH transaction belonging to the missing wWETH transfer.

**Reply by Citizen-000** [reply]
This is the ETH transaction: https://etherscan.io/tx/0x4cc020024e1b29103e26864d14b3f69efe14e2b628187d9416bebec6d988be85

**Reply by leoluk** [reply]
https://explorer.solana.com/tx/5ajdQo3WgwYQthPdTEJ6nxXLCwWH8FfPHxqPw1qLC3896RCZ1pcydwEqvTDsWkYt1xSjreeafVSMwzzi8bMtM4yP

**Reply by leoluk** [reply]
There you go!

**Reply by Citizen-000** [reply]
Thank you leo! Appreciate you
